src/Molecules/Notification/Notification.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import styled from 'styled-components'
+import { variant } from 'styled-system'
+import { createPortal } from 'react-dom'
+
+export const LEVEL_INFO = 'info'
+export const LEVEL_SUCCESS = 'success'
+export const LEVEL_ERROR = 'error'
+export const LEVEL_WARNING = 'warning'
+
+const Container = styled.div`
+  padding: 15px;
+  margin-bottom: 20px;
+  border: 1px solid transparent;
+  border-radius: 4px;
+  ${variant({ scale: 'levels' })}
+`
+
+const CloseIcon = styled.span`
+  justify-content: flex-end;
+  cursor: pointer;
+
+  font-size: 28px;
+  font-weight: normal;
+  line-height: 1;
+  text-shadow: 0 1px 0 #fff;
+`
+
+const Head = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+
+  & > h4 {
+    margin: 0;
+  }
+`
+
+const Notification = ({
+  children,
+  level = LEVEL_INFO,
+  show = false,
+  onClose = () => {},
+  title,
+  at = document.body,
+}) => (
+  !!show &&
+  createPortal(

```

Comment:
Same thing, the target is passed with *at* anyway, so better simply return a dumb component without knowledge of where it will be rendered. This way we can remove the portal, reducing complexity and coupling.

Reply:
This was a nice way to always have a target (which will be '#main-alert' in most cases). In the future maybe we would want a notification component without an `at` attribute.

Reply:
Well ok maybe it could be done on the dashboard project